### ISSUE TITLE
Server not working anymore? Is this project still maintained?

### ISSUE THREAD

**Issue by alonl** [oznu/homebridge-gsh#129]
**Describe The Bug:**
Getting timeouts while it keeps disconnecting from the server.

BTW, is this project still maintained? I see no commits in the last 6 months...

**To Reproduce:**
Just add the plugin.

**Expected behavior:**
It should automatically connect and sync with `homebridge-gsh` server, I believe.

**Logs:**

```
[31/08/2021, 20:37:08] [Google Smart Home] Error (23) - connect ETIMEDOUT 44.235.50.80:443
[31/08/2021, 20:37:08] [Google Smart Home] Disconnected (23) - Retry in 5000ms
[31/08/2021, 20:37:13] [Google Smart Home] Reconnecting (23)
[31/08/2021, 20:39:26] [Google Smart Home] Error (24) - connect ETIMEDOUT 54.71.233.216:443
[31/08/2021, 20:39:26] [Google Smart Home] Disconnected (24) - Retry in 5000ms
[31/08/2021, 20:39:31] [Google Smart Home] Reconnecting (24)
[31/08/2021, 20:41:44] [Google Smart Home] Error (25) - connect ETIMEDOUT 44.235.50.80:443
[31/08/2021, 20:41:44] [Google Smart Home] Disconnected (25) - Retry in 5000ms
[31/08/2021, 20:41:49] [Google Smart Home] Reconnecting (25)
[31/08/2021, 20:44:02] [Google Smart Home] Error (26) - connect ETIMEDOUT 54.71.233.216:443
[31/08/2021, 20:44:02] [Google Smart Home] Disconnected (26) - Retry in 5000ms
[31/08/2021, 20:44:07] [Google Smart Home] Reconnecting (26)
[31/08/2021, 20:44:08] [Google Smart Home] Connected (27) - wss://homebridge-gsh.iot.oz.nu
[31/08/2021, 20:48:10] [Google Smart Home] Disconnected (27) - Retry in 5000ms
[31/08/2021, 20:48:15] [Google Smart Home] Reconnecting (27)
```

**Homebridge Config:**

```json
{
    "bridge": {
        "name": "Homebridge FAA4",
        "username": "secret...",
        "port": 51973,
        "pin": "secret...",
        "advertiser": "bonjour-hap"
    },
    "accessories": [
    ],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "platform": "config"
        }
        {
            "name": "Google Smart Home",
            "token": "secret...",
            "notice": "Keep your token a secret!",
            "platform": "google-smarthome"
        }
    ]
}
```

**Screenshots:**
Irrelevant.

**Environment:**

* **Node.js Version**: v14.15.4
* **NPM Version**: v7.20.1
* **Homebridge Version**: v1.3.4
* **Homebridge GSH Plugin Version**: v2.0.1
* **Homebridge Config UI X Plugin Version**: v4.41.2
* **Operating System**: Raspbian

**Comment by Jacksonbm1** [comment]
I've also been having issues today. I tried reconnecting my google account to homebridge and it seems like it's just lagging out. Will post logs in a sec

**Comment by Jacksonbm1** [comment]
https://imgur.com/a/5NLh9rQ is an image of what I'm getting when trying to reconnect

Also, logs:

<google-sheets-html-origin><style type="text/css"></style>

[8/31/2021, 2:20:24 PM] [Google Smart Home] Finished instance discovery
--
[8/31/2021, 2:20:24 PM] [Google Smart Home] HAP Event listeners registered succesfully
[8/31/2021, 2:20:24 PM] [Google Smart Home] HAP Event listeners registered succesfully
[8/31/2021, 2:20:24 PM] [Google Smart Home] HAP Event listeners registered succesfully
[8/31/2021, 2:20:24 PM] [Google Smart Home] HAP Event listeners registered succesfully
[8/31/2021, 2:20:37 PM] [Google Smart Home] Sending State Report
[8/31/2021, 2:20:37 PM] [Google Smart Home] {
[8/31/2021, 2:20:39 PM] [Google Smart Home] Sending Sync Request
[8/31/2021, 2:21:17 PM] [Google Smart Home] Sending State Report
[8/31/2021, 2:21:17 PM] [Google Smart Home] {
[8/31/2021, 2:22:08 PM] [Google Smart Home] Sending State Report
[8/31/2021, 2:22:08 PM] [Google Smart Home] {
[8/31/2021, 2:22:12 PM] [Google Smart Home] Error (1) - connect ETIMEDOUT 44.235.50.80:443
[8/31/2021, 2:22:12 PM] [Google Smart Home] Disconnected (1) - Retry in 5000ms
[8/31/2021, 2:22:17 PM] [Google Smart Home] Reconnecting (1)
[8/31/2021, 2:22:48 PM] [Google Smart Home] Sending State Report
[8/31/2021, 2:22:48 PM] [Google Smart Home] {
[8/31/2021, 2:23:10 PM] [Google Smart Home] Sending State Report
[8/31/2021, 2:23:10 PM] [Google Smart Home] {
[8/31/2021, 2:23:19 PM] [Google Smart Home] Sending State Report
[8/31/2021, 2:23:19 PM] [Google Smart Home] {
[8/31/2021, 2:23:39 PM] [Google Smart Home] Sending State Report
[8/31/2021, 2:23:39 PM] [Google Smart Home] {
[8/31/2021, 2:24:09 PM] [Google Smart Home] Sending State Report
[8/31/2021, 2:24:09 PM] [Google Smart Home] {
[8/31/2021, 2:24:30 PM] [Google Smart Home] Sending State Report
[8/31/2021, 2:24:30 PM] [Google Smart Home] {
[8/31/2021, 2:24:30 PM] [Google Smart Home] Error (2) - connect ETIMEDOUT 44.235.50.80:443
[8/31/2021, 2:24:30 PM] [Google Smart Home] Disconnected (2) - Retry in 5000ms
[8/31/2021, 2:24:35 PM] [Google Smart Home] Reconnecting (2)
[8/31/2021, 2:25:00 PM] [Google Smart Home] Sending State Report
[8/31/2021, 2:25:00 PM] [Google Smart Home] {
[8/31/2021, 2:25:40 PM] [Google Smart Home] Sending State Report
[8/31/2021, 2:25:40 PM] [Google Smart Home] {
[8/31/2021, 2:26:11 PM] [Google Smart Home] Sending State Report
[8/31/2021, 2:26:11 PM] [Google Smart Home] {
[8/31/2021, 2:26:48 PM] [Google Smart Home] Error (3) - connect E

**Comment by vespino** [comment]
Don't think this project is maintained any longer. There have be bugs reported on the latest version which caused the entire system to not work any more, but no updates, not even a response from the developer. Are there alternative plugins?

**Comment by jotheman0303** [comment]
Such a shame it is no longer trustworthy, I've used this for around a year and it was great!
I used to use Controlicz to sync from Domoticz to Google Home, but this plugin made it unnecessary. Now, and only since a week or 2, it has become unreliable and Google keeps saying it cannot reach HomeBridge. Hope someone will "resuscitate" the project, but at least thanks to makers for having been able to use it over the past period!

To answer your question @vespino: Only alternative I found was the paid service called called Controlicz, which does the job though, just paid...

**Comment by WattageGuy** [comment]
Having the same problem. "cannot reach Homebridge". Sometimes it works and some times not. When it doesn't work I often see this in my hb log: Received QUERY intent. From my understadning this is a command realated to google smart home fulfillment url.

**Comment by Jacksonbm1** [comment]
Just want to respond to this: the project I'm fairly sure IS maintained. Last time this happened, I'm pretty sure they just added another server or something but the dev was asleep in Australia. Just give it a bit I think.

**Comment by Jacksonbm1** [comment]
I'd check out the discord btw, homebridge has one and sometimes you can see if the server itself is down.

**Comment by WattageGuy** [comment]
Thanks for the reply. According to discord it’s up running. Are you experiencing the same issue as we are?

**Comment by Jacksonbm1** [comment]
I wasn't home when the issue popped up, so I have no idea if it was happening to me. I am not experiencing any issues

**Comment by salimyzadeh** [comment]
I also get the same issue. Works for about a month and then times out. Removing google and re-adding in the config usually works but still only for a few months
[11/27/2021, 10:04:20 PM] [Google Smart Home] Reconnecting (39)
[11/27/2021, 10:06:33 PM] [Google Smart Home] Error (40) - connect ETIMEDOUT 44.240.224.238:443
[11/27/2021, 10:06:33 PM] [Google Smart Home] Disconnected (40) - Retry in 500

**Comment by vespino** [comment]
When I had problems I reverted to v2.0.1. When the update came out, I updated to v2.2.0. I'm currently not having any problems, thought they were back last week, so I rebuilt my docker container which seem to solve the whole thing. Might have tweaked too much in my install, so blamed myself. Running the latest version of all plugins.

**Comment by Micha-123** [comment]
Having the same problem. "cannot reach Homebridge". Sometimes it works and sometimes not


same here. whtas going wrong im on 2.2.0

**Comment by Jacksonbm1** [comment]
TL;DR, if you're reading this thread, check out discord before nuking your setup: https://discord.gg/Z8jmyvb

For everyone who keeps commenting on this thread. I'd strongly recommend that you check the discord. I'm sure that it is possible that there is an issue with the update. HOWEVER, I strongly suspect that it's usually a server error and that it just requires the dev to go do server maintenance or something (or it's just a temporary crash. For example @Micha-123, if you look at the discord log, you'll see that the server went down at 5:19 AM ET and went back up at 5:42, which is almost exactly when you were experiencing the issue. Nuking your setup might have been the solution, but I strongly suspect that it was just a temporary server error. Looking at the discord logs, it seems like the issues are short, but cluster, so if you do experience an issue, removing and re-adding might look like it's solving the issue, but it's also very possible you're just spending enough time doing something unrelated that it looks like it's fixing the problem.

If you do check discord and the issue turns out to not be transient (like you wait maybe an hour or two), then it might be a separate issue. Again, I really am not confident that nuking your setup is the right solution (though maybe there's a separate issue. If there is one, we should probably split this thread). Finally, note that the dev is in Australia, so their response may not be conducive to your time

Discord: https://discord.gg/Z8jmyvb

**Comment by vespino** [comment]
I haven’t had that many problems tbh. Sometimes “homebridge can’t be reached” and I reboot my Docker container and the issue is solved for a couple of weeks. So no complaints here.

**Comment by Micha-123** [comment]
By me it was realy a big problem, al the time google was realy slow to
communicate with homebridge and Sometimes i needed to ask it 2 or 3 times,
that isuse was all the time.

For now if i deleted the cashe in HB its working like a charm. If i ask
google it connects directly to HB and give annanswer without to "thinking"
about it. So i Guea my problem was the "full" cashe in HB. I habe realy
much devices

pon., 10 sty 2022, 16:03 użytkownik Jacksonbm1 ***@***.***>
napisał:

**Comment by Micha-123** [comment]
since yesterday much ETIMEOUTS, i see on discord that the GSH server went much times DOWN. 

i hope that wil be fixed. no answere on discord :(

**Comment by vespino** [comment]
I had the same issue, but the problem resolved itself. Could there have been issues with the Google Home network itself?

**Comment by marcgrc** [comment]
GSH server DOWN.

**Comment by marcgrc** [comment]
News ?

**Comment by vespino** [comment]
@marcgrc no issues here over the last couple of days.

**Comment by oznu** [comment]
I'm aware this service has had a few issues lately. I've finally gotten time to migrate the server component to new infrastructure that should handle the load more seamlessly.

**Comment by vespino** [comment]
Can we expect an update soon then?
